usr/tgtd.c

```diff
@@ -36,13 +36,16 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
-
+#include <stdbool.h>
```

Comment:
why do you need stdbool.h here

Reply:
needed for including TgtInterface.h

Reply:
as discussed moved stdbool include to tgtInterface.h file